App.tsx

```diff
@@ -1,59 +1,49 @@
-import React from 'react';
+import React, {useState} from 'react';
 import {SafeAreaView, View, FlatList, TouchableOpacity} from 'react-native';
 import Crypto from './src/components/Crypto';
 import {Container, Title, ImageApp, Space, BTNAddCrypto} from './styles';
 
-const cryptos = [
-  {
-    id: '1',
-    logo: require('./src/assets/images/bitcoin.jpg'),
-    name: 'Bitcoin',
-    abr: 'BTC',
-    price: 7215.68,
-    status: 1.82,
-  },
-  {
-    id: '2',
-    logo: require('./src/assets/images/et.png'),
-    name: 'Ethereum',
-    abr: 'ETH',
-    price: 146.83,
-    status: 1.46,
-  },
-  {
-    id: '3',
-    logo: require('./src/assets/images/xrp.png'),
-    name: 'XRP',
-    abr: 'XRP',
-    price: 0.220568,
-    status: -2.47,
-  },
-];
+const App = () => {
+  const [cryptos, setCryptos] = useState<any>([]);
```

Comment:
Don't use any type
create a type file to store the global types, in this case, the type for the crypto data:

src/
   types/
       crypto.ts